packages/atomic-layout-core/src/utils/functions/isServer/isServer.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Function that checks if the code is running on the node env or the client
+ * @returns {boolean}
+ */
+export default function isServer(): boolean {
```

Comment:
Feel free to remove this utility function and all related to it since we don't need to encapsulate this check.